package.json

```diff
@@ -48,7 +44,7 @@
     "babel-eslint": "^9.0.0",
     "babel-jest": "^23.4.2",
     "babel-loader": "^8.0.0",
-    "duti": "^0.14.0",
+    "duti": "^0.15.2",
     "eslint": "^5.6.1",
     "eslint-config-smartprocure": "^1.2.0",
     "futil": "^1.55.0",
```

Comment:
I know you didn't do this, but it made me realize - futil and lodash need to be dependencies, not dev dependencies

Reply:
They're peerDependencies

Reply:
looks like they're listed under `devDependencies` 🤔

Reply:
wait, disregard, they're in both